src/tokens/transformation/tailwind/twShadows.js

```diff
@@ -20,7 +20,9 @@ module.exports = {
  */
 const formatColorForTw = (colorString) => {
   const formattedColor = formatColor('tw', colorString)
-  return colorString.startsWith('$')
+  return colorString.startsWith('$') &&
+    !formattedColor.includes('elevation') &&
+    !formattedColor.startsWith('rgb')
```

Comment:
We don't do this for hex colors?

Reply:
Hmm... good question. I don't believe we're using HEX in Tailwind, though.

Reply:
Probably not a super useful check at any rate